### PR TITLE
Updated the deploy docs

### PR DESCRIPTION
**Pull request by thomasm-ttd**
Update the config for aws integration, azure and gcp

### CODE REVIEW

File: scripts/gcp/README.md [hunk 1]
```diff
@@ -133,139 +123,66 @@ write_files:
...
-as custom metadata under the key `user-data` when creating the VM instance. This `user-data`
-metadata will be read and interpreted by the Container-Optimized OS (COS) VM disk during
-booting.
-
-As shown in the template below, it first disables remote SSH access, and then tells
-COS VM to docker pull certified UID2 operator docker image from UID2 project's official
-Container Registry and run the UID2 operator container as a systemd service.
-
-```
-#cloud-config
```
Comment (scong-ttd) [inline scripts/gcp/README.md:200]: Do we need to keep the cloud-init here?

File: scripts/gcp/README.md [hunk 2]
```diff
@@ -112,11 +101,12 @@ write_files:
     [Service]
     Environment="UID2_ENCLAVE_API_TOKEN=<INPUT_API_TOKEN>"
     Environment="UID2_ENCLAVE_IMAGE_ID=<INPUT_IMAGE_ID>"
+    Environment="GHCR_RO_ACCESS_TOKEN=<GHCR_TOKEN>"
```
Comment (scong-ttd) [inline scripts/gcp/README.md:104]: any explanation what this token is and where for clients to get it?

Comment (thomasm-ttd) [reply scripts/gcp/README.md:104]: The clients don't get it, we will provide it. I will add a note about that

File: scripts/gcp/README.md (hunk 1 above)
Comment (thomasm-ttd) [reply scripts/gcp/README.md:200]: I thought having an example and discussing it was useful, but happy to remove the example if you don't think it is useful

Comment (scong-ttd) [reply scripts/gcp/README.md:200]: Oh I thought this was a deletion, but actually confusion from `git diff`.

**Review by scong-ttd — APPROVED** [review]

**Pull request merged**